### ISSUE TITLE
Uncaught ReferenceError: jsonx is not defined

### ISSUE THREAD

**Issue by mdlware** [repetere/jsonx#1055]
I tried the sample snippet from the docs in jsfiddle and it gives that error. Is this package still working or deprecated? 
Cheers!

**Comment by yawetse** [comment]
Closing as part of repository maintenance after #1088 landed on main with current dependency updates and passing tests/build. Please open a new issue with a minimal reproduction if this still reproduces on current main or the next published package.
